flink/v1.15/flink/src/main/java/org/apache/iceberg/flink/IcebergTableSink.java

```diff
@@ -37,6 +37,7 @@ public class IcebergTableSink implements DynamicTableSink, SupportsPartitioning,
   private final TableLoader tableLoader;
   private final TableSchema tableSchema;
   private final ReadableConfig readableConfig;
+  private final Map<String, String> tableProps;
```

Comment:
let's not call it `tableProps` to avoid confusing with Iceberg table properties. Is `writeProps` more accurate? it would be consistent with the Javadoc of `setAll` from `FlinkSink#Builder`.

```
    /**
     * Set the write properties for Flink sink. View the supported properties in {@link
     * FlinkWriteOptions}
     */
    public Builder setAll(Map<String, String> properties) {
      writeOptions.putAll(properties);
      return this;
    }
```

Reply:
done